src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/App_Plugins/UmbracoCms.Integrations/SEO/SemrushTools/statusEditor.html

```diff
@@ -0,0 +1,32 @@
+﻿<div ng-controller="UmbracoCms.Integrations.SemrushStatusController as vm">
+    <umb-editor-view>
+
+        <umb-editor-container>
+            <umb-box>
+                <umb-box-content>
+                    <p><b>Connected: {{ vm.IsAccessTokenAvailable }}</b></p>
```

Comment:
You might consider camel-casing here too - i.e. `vm.isAccessTokenAvailable.  As you've done on several of the DTOs, if you want to enforce this for data returned from API requests, you can add `[JsonProperty("myProperty")]` attributes.

Reply:
Item addressed.